### DISCUSSION TITLE
Accessibility Discussion Thread

### DISCUSSION THREAD

**Discussion by Fenrirthviti** [obsproject/obs-studio#6192] [General]
We've had a steady trickle of reports of things that are not ideal in OBS from an accessibility standpoint, so I'm creating a thread here to allow posting of any issues, or missing options, that OBS might have for accessibility. I will do my best to keep this list updated with any confirmed issues, linked PRs, issue posts, etc.

As an additional note, OBS does not have a VPAT, ACR, or similar accessibility report/certification. We unfortunately do not have anyone on the team familiar enough and with the appropriate skills/knowledge to pursue those at this time. If you are interested in helping out, feel free to reach out to us!

Current Confirmed Accessibility Concerns:

- https://github.com/obsproject/obs-studio/issues/6200
- https://github.com/obsproject/obs-studio/issues/5597
- https://github.com/obsproject/obs-studio/issues/13819

Unverified Reports (may be resolved/stale reports):

- https://github.com/obsproject/obs-studio/issues/3363
- https://github.com/obsproject/obs-studio/issues/3362
- https://github.com/obsproject/obs-studio/issues/3361
- https://github.com/obsproject/obs-studio/issues/2772
- https://github.com/obsproject/obs-studio/issues/2502

Open PRs:

N/A

Please use this thread to report any other issues or concerns with the *current UI in OBS*. Please avoid making additional feature requests, as those should be submitted to our [ideas page](https://ideas.obsproject.com)

**Comment by nebeker** [comment]
Issue #6200 should explain a number of accessibility issues that result from a non-standard UI design philosophy.

**Comment by Igor3947** [comment]
Hi,
I'm currently using NVDA on a Windows 10 desktop. I have noticed similar issues with regard to selecting sources; tabbing through the interface does not work, only the object navigator can detect the sources section and the add button. However, using the object navigator to activate the control causes the interface to become unresponsive. So, NVDA users have to use the move focus command in order to move the system focus to the object and then use space, or enter to activate the add button.
Another annoyance that plagues screen reader users is the remux feature. When trying to remux recordings, there is no way for me to select the video I want to convert. I am simply presented with a text box displaying the file name and data tables, where presumably I'm supposed to be able to drag and drop the target file; however, I cannot figure out how to do this.
If these issues are resolved, I will have no trouble with the rest of the application.

**Comment by Fenrirthviti** [comment]
Added #7614 as a reference that we will need to go through and correct any areas where the focus indicators are not working properly, and fix any consistency issues where they are currently showing. We are aware that the current state is not visually appealing, but as these are an important accessibility feature showing focus that is not conveyed in any other way, leaving them slightly ugly is better than removing them.

**Comment by RocksInMyDryer** [comment]
With only 1 monitor, using OBS as your main window is a necessity (so you can see the game & chat widget at once). Because of this, being able to change the sizing of the Docks' area (Scenes, Sources, Audio Mixer, Controls, etc) to make them small enough to see more of the game capture is super important! In the newest OBS release, the Docks are forcibly larger than ever, and can't be reduced. Allowing someone to make them smaller would be massively helpful, and have no real downside.

**Reply by Fenrirthviti** [reply]
You can set the theme back to Dark in Settings -> General.

**Comment by thedavegreen** [comment]
Hello, is there a VPAT or ACR available for OBS studio available for review?  Our District requires an alternative access plan to be developed for any software in use that has accessibility issues.  The VPAT and ACR are required to develop an alternative access plan.

**Reply by Fenrirthviti** [reply]
We unfortunately do not, I have updated the first post here to reflect the current status.
